src/text/parse_result.rs

```diff
@@ -70,9 +70,9 @@ pub(crate) type IonParseResult<'a, O> = IResult<&'a str, O, IonParseError<'a>>;
 /// // The same function as above, but spelled out a bit more explicitly
 /// fn verbose_recognize_boolean(input: &str) -> IonParseResult<&str> {
 ///     // Combine several core `nom` parsers to make a new parser with the desired logic
-///     let parser = terminated(          <-- Takes two parsers; if both succeed, returns the output of the first
-///         tag("true").or(tag("false")), <-- Parser 1 matches the text `true` or the text `false`
-///         tag(" ")                      <-- Parser 2 matches a single space
+///     let parser = terminated(          // <-- Takes two parsers; if both succeed, returns the output of the first
+///         tag("true").or(tag("false")), // <-- Parser 1 matches the text `true` or the text `false`
+///         tag(" ")                      // <-- Parser 2 matches a single space
```

Comment:
🗺️ Hazard of using `ignore`: the Rust code was not being compiled, so no one noticed that these lines should've been comments.